pkg/lib/cockpit-components-dialog.jsx

```diff
@@ -209,9 +209,8 @@ DialogFooter.propTypes = {
  *  - static_error optional, always show this error after the body element
  *  - footer (react element, top element should be of class modal-footer)
  *  - id optional, id that is assigned to the top level dialog node, but not the backdrop
- *  - variant: See PF4 Modal component's 'variant' property
- *  - titleIconVariant: See PF4 Modal component's 'titleIconVariant' property
- *  - showClose optional, specifies if 'X' button for closing the dialog is present
```

Comment:
`showClose` is no longer supported by PF, check all uses in cockpit & projects and drop this prop.

Reply:
found only one use in cockpit, other projects didn't use this prop at all.